lib/component.js

```diff
@@ -148,6 +148,31 @@ class Component extends WebComponent {
     this.$panelRoot.router.navigate(...arguments);
   }
 
+  /**
+   * Enqueue a function to be run at the end of the components connectedCallback. It can optionally return
+   * a function to run during the component's disconnectedCallback
```

Comment:
jsdoc for the auto-generated API docs is taken from here, not index.d.ts, so at least c/p the full version (with `@param` tag) here